### PR TITLE
add multiselect to document

### PR DESCRIPTION
**Pull request by jfrerich**
#### Summary
This PR adds the `multiselect` field name to the App Field documentation.  The relevant PR is here: https://github.com/mattermost/mattermost-plugin-apps/pull/55.  

The code documentation was added, but I just noticed the online docs change was missing. 

#### Ticket Link
n/a

### CODE REVIEW

File: site/content/integrate/apps/interactivity/_index.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ All fields include ([godoc](https://pkg.go.dev/github.com/mattermost/mattermost-
 | `name`        | string    | Key to use in the values field of the call.            |
 | `type`        | FieldType | The type of the field.                                 |
 | `is_required` | bool      | (Optional) Whether the field needs to be filled.       |
+| `multiselect` | bool      | (Optional) Whether a select field is multiselect.      |
```
Comment (hanzei) [inline site/content/integrate/apps/interactivity/_index.md:42]: 1/5 the sentence seems weird to me. Does it make sense to be more descriptive?

Comment (mickmister) [reply site/content/integrate/apps/interactivity/_index.md:42]: @hanzei Maybe something like "True when multiple values are able to be selected."

**Review by mickmister — CHANGES_REQUESTED** [review]
@jfrerich I'm not sure exactly what to put as the description, but just repeating the property name doesn't seem very useful. As I said in the comment above, it should convey something like "True when multiple values are able to be selected."

File: site/content/integrate/apps/interactivity/_index.md (hunk 1 above)
Comment (jfrerich) [reply site/content/integrate/apps/interactivity/_index.md:42]: I've thought through multiple variances of this description. The two things I think it should portray are 1) this is for select fields and 2) allows multiple values to be selected.  

The `bool` type lets the caller know that it is a true/false type.

How about `(Optional) Whether a select field allows multiple values to be selected.`?

**Review by hanzei — APPROVED** [review]

**Review by mickmister — APPROVED** [review]

**Pull request merged**